Dockerfile

```diff
@@ -1,10 +1,29 @@
-FROM rust:1.69.0-alpine as builder
-ARG BINARY=aggregator
-ARG GIT_REVISION=unknown
-RUN apk add libc-dev protobuf-dev protoc
+FROM rust:1.69.0-alpine AS chef
+RUN apk add libc-dev
```

Comment:
I split installation of those packages out into a later layer (in the builder phase) so that all docker builds can share the same layers up to installation of `cargo-chef`. (We should be able to drop them soon, once we can upgrade opentelemetry, but that requires shaving some other yaks first)